src/routes/[tripId]/+layout.server.ts

```diff
@@ -49,9 +65,10 @@ export const load = (async (event) => {
 		});
 	}
 
-	const rounds = roundsData.map(({ id, name, date }) => ({
+	const rounds = roundsData.map(({ id, name, date, courses }) => ({
 		id,
 		name,
+		courseName: Array.isArray(courses) ? courses[0]?.name : courses?.name,
```

Comment:
This is ugly, the supabase client joins don't seem to get the types perfect. This should only ever be one course but it's typed as possibly an array. Need to investigate.